src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/domain/AllocateCase.kt

```diff
@@ -7,5 +7,8 @@ data class AllocateCase @JsonCreator constructor(
   val crn: String,
   val eventId: BigInteger,
   val instructions: String = "",
-  val emailTo: List<String>? = null
-)
+  val emailTo: List<String>? = null,
+  val emailCopy: String? = null
+) {
+  fun sendEmailToAllocatingOfficer(): Boolean = emailCopy.isNullOrEmpty()
```

Comment:
I don't like this - there is nothing to indicate that the presence (or otherwise) of this property is the only way of determining whether it is true or false.

It really should be a boolean (and probably required).

Reply:
There aren't even any Swagger docs to make it more obvious!

Reply:
We did it like this because this is the default behaviour of checkboxes - otherwise we have to do some client side fiddling and we were trying to keep things as simple as possible in there. That sai there is already a lot of post processing of this form before it gets reposted. We can have a look into that - would move the complexity from hmpps-workload to mamage-a-workforce-ui.